SAMtemplates/main_template.yaml

```diff
@@ -0,0 +1,337 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  prescription status update api
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 50
+    MemorySize: 256
+    Architectures:
+      - x86_64
+    Runtime: nodejs18.x
+    Environment:
+      Variables:
+        TargetServiceSearchServer: !Ref TargetServiceSearchServer
+        NODE_OPTIONS: --enable-source-maps
+    Layers:
+      - !Sub "arn:aws:lambda:${AWS::Region}:580247275435:layer:LambdaInsightsExtension:38"
```

Comment:
current version is 49, do we want to bump this?

Reply:
The LambdaInsightsExtension version has been upgraded to 49